whelk-core/src/main/groovy/whelk/Whelk.groovy

```diff
@@ -32,15 +33,27 @@ class Whelk {
     ElasticSearch elastic
     SparqlUpdater sparqlUpdater
 
+    boolean completeCore = false
+
+    String applicationId
+    Map<String, Map<String, String>> namedApplications = [:]
+
+    String systemContextUri
+    String vocabUri
+    String vocabDisplayUri
+
+    ZoneId timezone = ZoneId.of('UTC')
+    List<String> locales = ['en']
```

Comment:
Perhaps make these mandatory in secret.properties (with no default)?

Reply:
We'll keep it for now (makes it easier to e.g. unit-test an unconfigured Whelk).